src/components/ContentSidebar/SkillsSidebar.js

```diff
@@ -174,7 +175,13 @@ class SkillsSidebar extends React.PureComponent<Props, State> {
         const { cards, errors }: State = this.state;
 
         return (
-            <SidebarContent title={<FormattedMessage {...messages.sidebarSkillsTitle} />}>
+            <SidebarContent
+                title={
+                    <SidebarTitle>
```

Comment:
Would naming this component `<SidebarContentTitle>` make where it's intended to be used more explicit? The semantics styling seems very specific to this area.